src/components/Kv/KvProgressCampaign.vue

```diff
@@ -2,46 +2,51 @@
 	<figure class="tw-grow">
 		<div class="tw-flex tw-items-center tw-justify-between">
 			<p>
-				{{ fundedBorrowers }} funded
+				{{ daysLeft }} days remaining
 			</p>
 			<p v-if="!minimalStats">
-				{{ borrowersLeft }} to go
+				{{ numeral(amountLeft).format('$0,0') }} to go
 			</p>
 		</div>
-		<kv-progress-bar
-			class="tw-my-0.5"
-			aria-label="Percent the loan has funded"
-			:value="progressPercentage"
-			:bg-variant="bgVariant"
-		/>
-		<div v-if="!minimalStats">
-			<p>
-				{{ daysLeft }} days left
-			</p>
+		<div class="tw-relative tw-mt-1">
+			<kv-progress-bar
+				class="tw-my-0.5"
+				aria-label="Percent the loan has funded"
+				:value="progressPercentage"
+				:bg-variant="bgVariant"
+			/>
+			<div class="tw-bg-white tw-rounded-full tw-absolute tw-right-0 tw-p-0.5">
+				<heart-out-from-box class="tw-w-3" />
+			</div>
 		</div>
 	</figure>
 </template>
 
 <script>
+import numeral from 'numeral';
+import HeartOutFromBox from '@/assets/inline-svgs/challenge/heart-out-from-box.svg';
 import KvProgressBar from '~/@kiva/kv-components/vue/KvProgressBar';
 
 export default {
 	name: 'KvProgressCampaign',
 	components: {
-		KvProgressBar
+		KvProgressBar,
+		HeartOutFromBox
 	},
 	props: {
-		fundedBorrowers: {
+		fundedAmount: {
```

Comment:
With these prop changes, make sure to remove it from the existing usage, `IwdProgressCampaign`. We still might be user testing the landing page, so we need to make sure it still loads, but the progress bar won't be shown without an active campaign, so we can remove it from that component.

Reply:
Let's keep that component for now, just comment out the `kv-progress-campaign`. We still don't know the future of that IWD page.